### ISSUE TITLE
Rename reports erroneous conflict for overloaded properties in VB

### ISSUE THREAD

**Issue by SLaks** [dotnet/roslyn#16576]
**Version Used**: 2017 RC Refresh

```VB
Public Class C
    Public ReadOnly Property P(a As Object) As Int32
        Get
            Return 2
        End Get
    End Property
    Public ReadOnly Property P2(a As String) As Int32
        Get
            Return P("")
        End Get
    End Property
End Class
```

Rename `P2` to `P`.
It will correctly add a cast to the callsite, but will also report an unresolvable conflict at the declaration of the original `P`, even though there's nothing wrong with that.

**Comment by JieCarolHu** [comment]
Still repro on the latest code, will look into it

![image](https://user-images.githubusercontent.com/17056651/43609098-ed007bbe-9657-11e8-93a0-f73a4039a61e.png)

**Comment by JoeRobich** [comment]
Fixed in 16.2
![image](https://user-images.githubusercontent.com/611219/60230859-31166b00-984c-11e9-9480-161fd32af96d.png)
